avm/res/sql/server/elastic-pool/main.bicep

```diff
@@ -39,17 +43,22 @@ param maintenanceConfigurationId string?
 @description('Optional. The storage limit for the database elastic pool in bytes.')
 param maxSizeBytes int = 34359738368
 
+// TODO: Only for Serverless
```

Comment:
Leftover?

Reply:
I was not sure why the description specifically mentions the serverless pool. I had the impression that for elastic pool you need a provisioned server. Anyhow, let's figure out later.